tests/integration/models/reddit/test_comment.py

```diff
@@ -153,9 +153,7 @@ def test_parent__submission(self):
     def test_permalink(self):
         with self.recorder.use_cassette('TestComment.test_permalink'):
             comment = Comment(self.reddit, 'cklhv0f')
-            assert comment.permalink() == ('/r/redditdev/comments/2gmzqe/'
-                                           'praw_https_enabled_praw_testing_'
-                                           'needed/cklhv0f')
+            assert comment.permalink() == '/comments/2gmzqe//cklhv0f'
```

Comment:
Is this test accurate? With the new permalink method, all permalinks are of the "non-fast" type, so the "fast" type, which you check here, should never be returned.